### DISCUSSION TITLE
link username

### DISCUSSION THREAD

**Discussion by powquad** [community/community#66824] [Other Feature Feedback, Questions, & Ideas]
how do I change the address name of my github page?

**Comment by andreslopezferro** [comment]
Hey!
Your github url is linked directly to your username. So to change your username on GitHub, you can follow these steps:

1. Sign in to GitHub:
- Go to the GitHub website (https://github.com/) and sign in to your account if you're not already signed in.

2. Access Your Account Settings:

- Click on your profile picture or your current username in the top right corner of the page.
- In the dropdown menu, select "Settings."

3. Change Your Username:

- In the left sidebar, click on "Account"
- Under the "Change username" section, you'll see a "Change username" button.
- Click on the "Change username" button and choose a new user name!

Hope it helps.

**Comment by thanmaisai** [comment]
Certainly! Here are concise steps to change the repository name, username, and set up a custom domain for your GitHub Pages repo:

**Changing Repository Name:**

1. **Log in to GitHub.**
2. **Access Your Repository:** Open the repository you want to rename.
3. **Go to Settings:** Click the "Settings" tab within the repository.
4. **Change Repository Name:** In the "Options" section, edit the "Repository name." Follow naming rules.
5. **Update Links:** Update any links pointing to your repository.
6. **Save Changes:** Scroll down and click "Save changes."

**Changing Username:**

1. **Log in to GitHub.**
2. **Access Account Settings:** Click your profile picture and select "Settings."
3. **Change Username:** In the "Account settings" section, change your username.
4. **Confirm Password:** You'll need to enter your password.
5. **Update Links:** Update links referencing your GitHub account.

[**Setting Up Custom Domain:**](https://www.youtube.com/watch?v=rIXWUJ5U8bY)

1. **Buy a Domain:** Purchase a domain from a domain registrar.
2. **DNS Configuration:** Set up DNS records (A or CNAME) pointing to GitHub Pages servers. Consult your registrar's documentation.
3. **Repository Settings:** In your GitHub Pages repository, go to "Settings."
4. **Custom Domain:** In the "[Custom domain](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site)" section, enter your domain (e.g., `www.yourdomain.com`) and save.
5. **DNS Verification:** Add a CNAME record with your domain registrar to verify ownership.
6. **Wait for Propagation:** DNS changes may take some time to propagate.
7. **GitHub Pages with Custom Domain:** Your site will be accessible at your custom domain.

## **Remember to update any references and links to your GitHub Pages site after making these changes.**

**Comment by ifsvivek** [comment]
You can change the address name of your GitHub Page by configuring a custom domain. To do this, follow these steps:

1. Go to the repository settings for the repository that you want to use as your GitHub Pages site.
2. In the "Code and automation" section, click on "Pages".
3. Under "Custom domain", type the domain name that you want to use.
4. Click on "Save".

Once you have configured a custom domain, you will need to update your DNS records to point to GitHub Pages. You can find instructions on how to do this on the GitHub Pages documentation: [How to configure a custom domain for GitHub Pages](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site)


It may take some time for the changes to propagate, so be patient. Once the changes have propagated, you should be able to access your GitHub Pages site at the custom domain that you specified.

Here are some additional things to keep in mind:

* You can only use a custom domain for a GitHub Pages site if you have a public repository.
* You can only use a custom domain that is not already in use by another website.
* You may need to contact your domain registrar to update your DNS records.

I hope this helps!
